joomla/joomla-cms
# [4.0] Fix overrides language strings

### Summary of Changes
Change/match constant keys to values.
Fix sort dropdown to display correctly when sorting by text.


### Testing Instructions
Go to System > Language Overrides
Create overrides
Click Text header to sort
See `Sort Table by` dropdown not showing the selection
